src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.figures;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int upperBase;
+    private int lowerBase;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, String name, int upperBase, int lowerBase, int height) {
+        super(color, name);
+        this.upperBase = upperBase;
+        this.lowerBase = lowerBase;
+        this.height = height;
```

Comment:
The `name` parameter is being passed to the superclass constructor. Ensure that this is necessary, as the superclass might already handle the `name` property. If it's redundant, consider removing it.